common/views/components/SearchFilters/SearchFiltersMobile.js

```diff
@@ -105,7 +105,11 @@ const ActiveFilters = styled(Space).attrs({
 
 const FiltersBody = styled(Space).attrs({
   h: { size: 'xl', properties: ['padding-left', 'padding-right'] },
-})``;
+})`
+  input[type='number'] {
+    min-width: calc(24px + 4ch);
```

Comment:
Just wondering why we've decided to go with this say over `px`? Is it just lucky that the characters in these boxes are ~ the `0` width?

Reply:
I was thinking of it as 4 digits, so the `ch` unit seemed appropriate. Although I suppose you're right – it is lucky that 'Year' has narrow-ish characters.